build.fsx

```diff
@@ -1,15 +1,23 @@
 #r "packages/FAKE/tools/FakeLib.dll" // include Fake lib
+#r "System.Management"
 
 open System
 open System.Net
 open System.IO
-
+open System.Management
 open Fake
 open Fake.Git
+open Fake.ProcessHelper
 
 let releaseRepo = "https://github.com/guillermooo/sublime-fsharp-package-releases.git"
 
-let sublimePath () =
+// build parameters
+let dataDir     = getBuildParam "sublimeDir"
```

Comment:
It is aligned against releaseRepo a few lines ago. They were closer together previously.
If there are several assignments, I like to align the `=` for readability.